belt-block/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+//! Pure Rust implementation of the [Belt-block][belt-block]
+//!
+//! # ⚠️ Security Warning: Hazmat!
+//!
+//! This crate implements only the low-level block cipher function, and is intended
+//! for use for implementing higher-level constructions *only*. It is NOT
+//! intended for direct use in applications.
+//!
+//! USE AT YOUR OWN RISK!
+//!
+//! [belt-block]: https://ru.wikipedia.org/wiki/BelT
+
+pub use cipher;
+
+mod consts;
+mod block;
+
+pub use crate::block::BeltBlock;
```

Comment:
I don't think it's worth to introduce the `block` module. You can simply move its code here.

Reply:
It was made bcs i'm planning to add wblock too :)
Need to remove it by now?

Reply:
Hm, IIUC `belt-wblock` looks closer to a block mode, so I am not sure it's worth to define it in this crate. We could either introduce it in a separate crate or, since it a relatively simple algorithm, have private implementations in the higher level algorithms which use it.

Reply:
Okay, as you say, thanks :)